pkg/lib/form-layout.less

```diff
@@ -43,8 +47,17 @@
 // a <form class="ct-form-layout"> and layout magic happens.
 
 .ct-form-layout {
+  // Locally redefine padding back to Bootstrap values
+  @padding-large-vertical: 10px;
+  @padding-small-vertical: 5px;
+  @padding-small-horizontal: 10px;
```

Comment:
I tested the redefinitions to be absolutely sure they do not leak out to the rest of Cockpit.

LESS is block-scoped, and the responsive block of CSS at the bottom of this file that is _not_ nested under `.ct-form-layout` does not use these values. It uses PatternFly's global redefinition, not the local redefinitions here, which are values based on Bootstrap. (This means the deconstructed mobile version uses slightly different values than the mods here. But the rest should be 100% the same.)

These values are what I used for the grid layout and map 1:1 to the values previously used, hence re-using them here.